### ISSUE TITLE
Fix keyboard accessibility of uberbar menu in manager (accessibility defect)

### ISSUE THREAD

**Issue by paulbohman** [modxcms/revolution#12151]
For compliance with Web Content Accessibility Guidelines (WCAG)

The main drop down menus are not accessible by keyboard. This is a big problem -- it's a deal-breaker, really -- that affects blind users and sighted keyboard users, neither of which can use a mouse effectively. The menu is 100% inaccessible to these groups. The ideal would be to use an ARIA-based menu that allows users to tab into the group (the whole nav/menu group) and then tab past it without tabbing into the menu items at all, and use the arrow keys to navigate within the menu system itself. See the keyboard patterns outlined here:

http://www.w3.org/TR/wai-aria-practices/#menu
http://www.w3.org/TR/wai-aria-practices/#kbd_focus

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#uberbar

**Comment by rthrash** [comment]
This is being addressed at the [MODX Accessible CMS Project](https://github.com/modxcms/a11y/issues/), and with Paul's agreement has been moved there. For more details, see https://github.com/modxcms/revolution/issues/12156#issuecomment-72382480. Thank you!
